droidlet/dashboard/web/src/MainPane.js

```diff
@@ -15,47 +16,71 @@ class MainPane extends React.Component {
       depth: null,
       objects: null,
       humans: null,
+      agentType: 'locobot',
     };
   }
 
+  componentDidMount() {
+    if (this.props.stateManager) this.props.stateManager.connect(this);
+  }
+
   render() {
+    console.log("MainPane rendering agent type: " + this.state.agentType);
     const stateManager = this.props.stateManager;
 
-    return (
-      <div>
-        <InteractApp stateManager={stateManager} />
-        <LiveImage
-          type={"rgb"}
-          height={320}
-          width={320}
-          offsetH={320 + 80}
-          offsetW={10}
-          stateManager={stateManager}
-        />
-        <LiveImage
-          type={"depth"}
-          height={320}
-          width={320}
-          offsetH={320 + 80}
-          offsetW={10 + 320 + 10}
-          stateManager={stateManager}
-        />
-        <LiveObjects
-          height={320}
-          width={320}
-          offsetH={320 + 60 + 320 + 30}
-          offsetW={10}
-          stateManager={stateManager}
-        />
-        <LiveHumans
-          height={320}
-          width={320}
-          offsetH={320 + 60 + 320 + 30}
-          offsetW={10 + 320 + 10}
-          stateManager={stateManager}
-        />
-      </div>
-    );
+    if (this.state.agentType === 'locobot'){
```

Comment:
Gotcha, could I just change the default agentType value to null?

Reply:
Actually I'm realizing it may not be as straight forward to show a loading indicator here. 

Usually the component is making the api call for the data it needs (so it knows the loading state), but that's not happening here. 

I think for now this is ok. We can explore solutions for loading down the road.